### PR TITLE
Translation: Raw TCP Traffic Shaping with Istio 1.1.0

### PR DESCRIPTION
**Pull request by malphi**
翻译的文档的 Issue 编号是：https://github.com/servicemesher/trans/issues/134
NOTE：
文章标题shaping with 意为“塑造、加工”，修饰“流量”并不合适，故翻译为“控制”；
文章中TCP ECHO Server，server在此处翻译为服务端、服务器都不贴切，故翻译为服务。

### CODE REVIEW

File: 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+[1] 69266
+Forwarding from 127.0.0.1:9000 -> 9000
+Forwarding from [::1]:9000 -> 9000
+```
+
+现在可以发送一些请求到加权负载均衡的TCP Echo服务：
+
+```bash
+$ for i in {1..10}; do
+for> docker run -it --rm busybox sh -c '(date; sleep 1) | nc docker.for.mac.localhost 9000'
```
Comment (venilnoronha) [inline 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md]: Replace `docker.for.mac.localhost 9000` with `192.168.99.100 31400` after identifying the ingress IP. See https://github.com/venilnoronha/venilnoronha.github.io/blob/source/_posts/2018-10-19-raw-tcp-traffic-shaping-with-istio-1.1.0.md#verifying-istios-tcp-routing-behavior.

File: 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+![Architecture](https://ws2.sinaimg.cn/large/006tNbRwly1fwhdkuu6sgj30zg0ikmze.jpg)
+
+## 清理
+
+首先，通过下面的命令停止端口转发：
+
+```bash
+$ killall kubectl
+[1]  + 69266 terminated  kubectl port-forward istio-ingressgateway-7b9bff766d-l478h -n istio-system
+```
```
Comment (venilnoronha) [inline 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md]: This step was removed: https://github.com/venilnoronha/venilnoronha.github.io/blob/source/_posts/2018-10-19-raw-tcp-traffic-shaping-with-istio-1.1.0.md#cleanup

File: 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 9000
```
Comment (venilnoronha) [inline 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md]: Also, this has been updated: https://github.com/venilnoronha/venilnoronha.github.io/blob/source/_posts/2018-10-19-raw-tcp-traffic-shaping-with-istio-1.1.0.md#routing-configuration

**Comment by venilnoronha** [comment]
GitHub has finally published these changes to my blog: https://venilnoronha.io/raw-tcp-traffic-shaping-with-istio-1.1.0

**Comment by malphi** [comment]
> GitHub has finally published these changes to my blog: https://venilnoronha.io/raw-tcp-traffic-shaping-with-istio-1.1.0

OK，I will follow your latest post to translate.

File: 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+...
+Kubectl is now configured to use the cluster.
+Loading cached images from config file.
+```
+
+### 安装 Istio
+
+在撰写本文时，Istio 1.1.0还没有发布。因此我使用了Istio的 [Daily Pre-Release](https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/master-20181017-09-15/)来演示这个新特性。请参考[Istio文档](https://istio.io/docs/setup/kubernetes/download-release/)学习下载和配置Istio。
+
+一旦部署完毕，有一个简单的方法来完全部署Istio：
```
Comment (rootsongjc) [inline 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md]: 这句意思不对吧。

File: 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+$ kubectl apply -f install/kubernetes/istio-demo.yaml
+namespace/istio-system created
+...
+destinationrule.networking.istio.io/istio-policy created
+destinationrule.networking.istio.io/istio-telemetry created 
+```
+
+### 使用Istio代理部署TCP Echo服务
+
+为了演示Istio的路由机制，我们以side-car模式部署`tcp-echo-server` ：
```
Comment (rootsongjc) [inline 201810/raw-tcp-traffic-shaping-with-istio-1.1.0.md]: 一般写作Sidecar，不需要连字符

**Pull request merged**